frontend/components/menu/popup/component.tsx

```diff
@@ -55,39 +57,47 @@ export const Popup: React.FC<PopupProps> = ({
     <FocusScope restoreFocus>
       <div {...overlayProps} ref={overlayRef}>
         <DismissButton onDismiss={onClose} />
-        <ul
+        <div
           {...mergeProps(menuProps, domProps)}
           ref={ref}
           className={cx(
             'z-30 absolute transform whitespace-nowrap bg-white shadow-2xl rounded-md overflow-hidden',
             align === 'start' ? 'left-0' : 'right-0',
             direction === 'top' ? '-top-2 -translate-y-full' : '-bottom-2 translate-y-full'
           )}
+          aria-labelledby="popup-header"
```

Comment:
🟠

Seeing this in action, I think that was a bad idea to tell you to add this attribute. Can you remove it?

```suggestion
```